SysCache2/NHibernate.Caches.SysCache2.Tests/SysCacheProviderNoConfigFixture.cs

```diff
@@ -0,0 +1,71 @@
+#region License
+
+//
+//  SysCache - A cache provider for NHibernate using System.Web.Caching.Cache.
+//
+//  This library is free software; you can redistribute it and/or
+//  modify it under the terms of the GNU Lesser General Public
+//  License as published by the Free Software Foundation; either
+//  version 2.1 of the License, or (at your option) any later version.
+//
+//  This library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//  Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public
+//  License along with this library; if not, write to the Free Software
+//  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+//
+// CLOVER:OFF
+//
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using NHibernate.Cache;
+using NHibernate.Caches.Common.Tests;
+using NUnit.Framework;
+
+namespace NHibernate.Caches.SysCache2.Tests
+{
+	[TestFixture]
+	public class SysCacheProviderNoConfigurationFixture
```

Comment:
The cache provider still have regions with this test. This test does never fail, unless removing manually the config file after compilation. There are no way to test this indeed, if we want to keep on testing the nominal case (having regions configured) without fiddling with CI scripts. (We could otherwise launch a second test run in CI after having deleted the configuration file.)